flexible/cloudsql/src/main/resources/config.properties

```diff
@@ -11,5 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+# [START properties]
```

Comment:
This doesn't look like it's actually showing that much.  What are you teaching here?